### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by gravelfreeman** [sereneblue/chameleon#599] [General]
I just want to say thank you for this awesome piece of work. I was able to get rid of 5 extensions and keep UBO, Chameleon and Multi-Account Containers. My browser is much faster and websites I've tried so far are working. I'm defeating websites like Amiunique with the plugins I just enumerated plus a proxy in my containers.

While I'm here, I would like to know if there's a way to persist custom profile configurations per individual containers?

This would be a great feature to fake and persist an identity per container. Then whenever I want to fake an hardware change, I could press the Chameleon button and that new identity would persist in that container.

**Comment by sereneblue** [comment]
Hi @gravelfreeman,

Thank you for the kind words. Chameleon does not currently support containers. This feature has been suggested in the past and is something I'm considering adding support for in the Manifest V3 version of Chameleon. When Mozilla releases a timeline to deprecate Manifest V2 extensions, I'll provide some more detail about the status of this feature.
